app/views/books/copies/new.html.erb

```diff
@@ -0,0 +1,11 @@
+Create a new copy here!
+
+<%= form_for [@book, @book.copies.build] do |f| %>
+  <%= f.label(:book_id) %>
+  <%= f.text_field(:book_id) %>
+  <%= f.label(:borrower) %>
+  <%= f.text_field(:borrower) %>
+  <%= f.label(:due_date) %>
+  <%= f.text_field(:due_date) %>
+  <%= submit_tag("Submit") %>
+<% end %>
```

Comment:
You can save duplicate code by using a partial view between this and your edit.html.erb file